### DISCUSSION TITLE
FV boundary condition - not registered

### DISCUSSION THREAD

**Discussion by heinono1** [idaholab/moose#23242] [Q&A General] [answered]
Hi. I am testing some FV modeling. I am using a simple input file with only FVDirichlet and FVTimeKernel, and FVDirichletBC. Running the code throws an error right away: "Task add_bc is not registered to build FVBoundaryCondition derived objects" I am a little puzzled by this: are not all FV objects in the main trunk repo registered to be built in a standard moose build?

Thanks,
Olle

**Comment by GiudGiud** [comment, marked as answer]
Hello

It should not be in the [BCs] block, it should be in the [FVBCs] block

Guillaume

**Comment by heinono1** [comment]
Hi Guillaume,

Haha I should have guessed that. Should all relevant blocks (Kernels, AuxKernels,...) have the FV prefix?

Thanks,
Olle

**Reply by GiudGiud** [reply]
FVKernels yes

AuxKernels works though

**Reply by heinono1** [reply]
OK, thanks.
